packages/react/src/hooks/contracts/useContractReads.ts

```diff
@@ -18,9 +22,15 @@ export type UseContractReadsConfig<
   [K in keyof Config]?: K extends 'contracts'
     ? DeepPartial<Config[K], 2>
     : Config[K]
-} & QueryConfig<ReadContractsResult<TContracts>, Error, TSelectData> & {
+} & QueryConfigWithSelect<
+  ReadContractsResult<TContracts>,
+  Error,
+  TSelectData
+> & {
     /** If set to `true`, the cache will depend on the block number */
     cacheOnBlock?: boolean
+    /** Set this to `true` to keep the previous data when fetching based on a new query key. Defaults to `false`. */
+    keepPreviousData?: boolean
```

Comment:
One-off fix. Will be refactored for `wagmi@1`, where we expose all query options.